Docs/debugging.md

```diff
@@ -0,0 +1,76 @@
+# Debugging
+
+The content of this guide will help you to configure your PHPStorm in order to debug a PHP Application.
+The following doc is tested with Linux, but be adapted for other host systems.
+
+## Container config
+
+*Xdebug* is deactivated by default. If you want to activate it, you need to set the following environment variables.
+
+In this case we use a part of the default configuration of a php-fpm and httpd application but this is the same spirit for others versions:
```

Comment:
`this is in the same vein`
"the same spirit" is a french construction ;-)

Reply:
Yeah, When I wrote it, I was wondering if it was really English :D